app/artifact-cas/internal/conf/conf.proto

```diff
@@ -88,4 +89,9 @@ message Credentials {
       // mount path of the kv engine, default /secret
       string mount_path = 3;
     }
+
+    message GCPSecretManager {
+        string project_id = 1;
+        string auth_key   = 2;
```

Comment:
Actually, when doing the changes around examples I thought about providing the key or key location and changed the code to use the location of the key. I also did the naming change to be descriptive, so that the user doesn't have to think about the options, and comments were added as well.